### DISCUSSION TITLE
Reboot command

### DISCUSSION THREAD

**Discussion by jamesholcomb** [dgreif/ring#1341] [Q&A]
Does a reboot command exist? My Ring Doorbell Pro has the stuck IR cut filter problem (i.e. pink effect) and rebooting seems to solve it for the day.

**Comment by dgreif** [comment]
I haven't see this anywhere. Are you able to reboot it via the Ring app?

**Comment by jamesholcomb** [comment]
Yes, from the Device Health menu.
